docs/installation/server-config/index.rst

```diff
@@ -29,70 +31,71 @@ For example:
 
-.. _restrict-ksql-interactive:
 
-Restricting Interactive KSQL Usage
-----------------------------------
+.. _restrict-ksql-interactive:
 
-KSQL supports locked-down deployment scenarios where you can restrict interactive use of the KSQL cluster.
+-------------------------------------
+Non-interactive (Headless) KSQL Usage
+-------------------------------------
 
-You can prevent interactive use of a KSQL cluster. For example, you want to allow a team of users to develop and verify
-their queries on a shared testing KSQL cluster. But when putting those queries to production you prefer to lock-down access
-to KSQL servers, version-control the exact queries and storing them in a .sql file, and prevent users from interacting
+KSQL supports locked-down, "headless" deployment scenarios where interactive use of the KSQL cluster is disabled.
+For example, to allow a team of users to develop and verify their queries interactively on a shared testing
+KSQL cluster. But when deploying those queries in your production environment, you want to lock-down access to KSQL
+servers, version-control the exact queries, and store them in a .sql file. This will prevent users from interacting
 directly with the production KSQL cluster.
 
-You can configure servers to run a predefined script (.sql file) via the ``--queries-file`` command line argument, or the
-``ksql.queries.file`` setting in the :ref:`KSQL configuration file <common-configs>`. If a server is running a predefined
-script, it will automatically disable its REST endpoint and interactive use.
+You can configure servers to exclusively run a predefined script (``.sql`` file) via the ``--queries-file`` command
+line argument, or the ``ksql.queries.file`` setting in the :ref:`KSQL configuration file <ksql-server-config>`. If a
+server is running a predefined script, it will automatically disable its REST endpoint and interactive use.
 
-.. tip::
-If the ``ksql.queries.file`` property and the ``--queries-file`` argument are present, the ``--queries-file`` argument will take precedence.
+.. tip:: When both the ``ksql.queries.file`` property and the ``--queries-file`` argument are present, the ``--queries-file`` argument will take precedence.
 
-Start the KSQL server in via the command line argument
+To start the KSQL Server in headless, non-interactive configuration via the ``--queries-file`` command line argument:
     #. Create a predefined script and save as an ``.sql`` file.
 
     #. Start the KSQL with the predefined script specified via the ``--queries-file`` argument.
 
        .. code:: bash
 
             $ <path-to-confluent>/bin/ksql-start-server <path-to-confluent>/etc/ksql/ksql-server.properties \
-              --queries-file <path-to-queries-file>.sql
+              --queries-file /path/to/queries.sql
 
-Start the KSQL server in via the ``ksql-server.properties`` file
+To start the KSQL Server in headless, non-interactive configuration via the ``ksql.queries.file`` in the server
```

Comment:
this line break after `server` is causing the definition list to lose its formatting: 
![image](https://user-images.githubusercontent.com/11722533/38274125-a2b30ae6-3742-11e8-91cd-506ae86d94ef.png)

Should be a single non-broken line: `To start the KSQL Server in headless, non-interactive configuration via the ``ksql.queries.file`` in the server configuration file:`